ArduinoController/ArduinoController.ino

```diff
@@ -10,43 +10,41 @@
 #include "StartController.h"
 #include "PinJSON.h"
 
-const uint8_t KILLPIN = 50; // Needs to remain global (therefore outside of JSON) for lambda interrupt
-const uint8_t KILL_ADDR = 10;
 const uint32_t CONTROLLER_CNT = 13u;
 class IController* controllers[CONTROLLER_CNT];
 
 void setup() {
   Serial.begin(115200);
-  Serial.print('R');
-  DynamicJsonBuffer buffer;
-  auto json = buffer.parse(PinJSON::json);
```

Comment:
Why did you remove the JSON? One of our goals is to be able to reconfigure the Arduino pin map live from the Beaglebone by sending new JSON. This moves in the opposite direction from that.